### PR TITLE
Display a single job offer

### PR DESCRIPTION
**Pull request by ianrodrigues**

### CODE REVIEW

File: src/Entity/JobOffer.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\JobOfferRepository")
+ */
+final class JobOffer
+{
+    const STATUS_PENDING_REVIEW = 'PENDING_REVIEW';
```
Comment (rtio) [inline src/Entity/JobOffer.php]: Should we use an Enum here?

File: src/Entity/JobOffer.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $company;
```
Comment (rtio) [inline src/Entity/JobOffer.php]: Will it becomes a future constraint?

File: src/Entity/JobOffer.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=10)
+     */
+    private $seniorityLevel;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $salary;
```
Comment (rtio) [inline src/Entity/JobOffer.php]: Money type? May float?

File: src/Entity/JobOffer.php (hunk 1 above)
Comment (ianrodrigues) [reply src/Entity/JobOffer.php]: It's possible.

File: src/Entity/JobOffer.php (hunk 2 above)
Comment (ianrodrigues) [reply src/Entity/JobOffer.php]: Yes, for now, we can work with strings.

File: src/Entity/JobOffer.php (hunk 3 above)
Comment (ianrodrigues) [reply src/Entity/JobOffer.php]: Yes, we can improve that. I just started with bare minimum, we start with other cards. We can create a card for that.

Will we use a single value or a range like:

* 3500.49
Or:
* 2500.99 - 3000.05

**Comment by ianrodrigues** [comment]
It closes #14

**Pull request merged**